static/app/components/sidebar/onboardingStatus.tsx

```diff
@@ -116,7 +116,7 @@ function OnboardingStatus({
 const Heading = styled('div')`
   transition: color 100ms;
   font-size: ${p => p.theme.backgroundSecondary};
-  color: ${p => p.theme.gray200};
+  color: ${p => p.theme.subText};
```

Comment:
Hmm, but now the main text and subtext are both subtext? Can we just use normal text color here or something?

Reply:
We could use `white` here (cause the sidebar has a dark background in both light and dark mode). Didn't use it cause I thought it would make the text stand out too much. But ehh we want people to notice this anyway.

<img width="215" alt="Screen Shot 2021-12-16 at 3 55 23 PM" src="https://user-images.githubusercontent.com/44172267/146466103-74537505-c635-4237-ba1f-637600c0ec8f.png">